bundles/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/PlugwiseCommunicationContext.java

```diff
@@ -97,8 +96,20 @@ public void closeSerialPort() {
         SerialPort localSerialPort = serialPort;
         if (localSerialPort != null) {
             try {
-                IOUtils.closeQuietly(localSerialPort.getInputStream());
-                IOUtils.closeQuietly(localSerialPort.getOutputStream());
+                if (localSerialPort.getInputStream() != null) {
```

Comment:
I would suggest to assign a local variable here

Reply:
Done